sql/information_schema/information_schema.go

```diff
@@ -1421,7 +1421,14 @@ func statisticsRowIter(ctx *Context, c Catalog) (RowIter, error) {
 							collation = "A"
 
 							// TODO : cardinality is an estimate of the number of unique values in the index.
-
+//							cardinality = 10000
+
+							if st, ok := index.(StatisticsIndex); ok {
+                               		                cardinality = st.RowCount()
```

Comment:
Make sure you're using `go fmt` to format all your code. I see a few places where formatting is off.